src/Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights/Converters/DependencyTelemetryConverter.cs

```diff
@@ -20,15 +32,26 @@ protected override DependencyTelemetry CreateTelemetryEntry(LogEvent logEvent, I
             var resultCode = logEvent.Properties.GetAsRawString(ContextProperties.DependencyTracking.ResultCode);
             var outcome = logEvent.Properties.GetAsBool(ContextProperties.DependencyTracking.IsSuccessful);
             var operationId = logEvent.Properties.GetAsRawString(ContextProperties.Correlation.OperationId);
+            var eventContext = logEvent.Properties.GetAsDictionary(ContextProperties.EventTracking.EventContext);
 
             var dependencyTelemetry = new DependencyTelemetry(DependencyType.ToString(), target, dependencyName, data, startTime, duration, resultCode, success: outcome)
             {
                 Id = operationId
             };
 
+            foreach (KeyValuePair<ScalarValue, LogEventPropertyValue> contextProperty in eventContext)
```

Comment:
Several converters were missing the properties conversion from the `LogEvent` to a `ITelemetry` instance.